url/interfaces.any.js

```diff
@@ -28,7 +21,7 @@
   USVString toJSON();
 };
 
-[Constructor(optional (USVString or URLSearchParams) init = ""),
+[Constructor(optional (sequence<sequence<USVString>> or USVString) init = ""), // or record<USVString, USVString>
```

Comment:
Not sure, https://github.com/w3c/webidl2.js/pull/57 supposedly added record support. I get "Unterminated generic type record, line 20 (tokens: ', USVString> ')" when I include the commented out line in the IDL.